pkg/cmd/kafka/create/create.go

```diff
@@ -255,7 +256,7 @@ func runCreate(opts *Options) error {
 
 	if !opts.wait {
 		opts.Logger.Info()
-		opts.Logger.Info(opts.localizer.MustLocalize("kafka.create.info.successAsync", nameTemplateEntry))
+		opts.Logger.Info(icon.Success(), opts.localizer.MustLocalize("kafka.create.info.successAsync", nameTemplateEntry))
```

Comment:
I wouldn't put the icon here, because the Kafka instance is being created, and could fail, so the icon might be a false indicator.